docs/data-sources/aviatrix_network_domains.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "TGW Orchestrator"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_network_domains"
+description: |-
+  Get a list of all Network Domains and detail
+---
+
+# aviatrix_all_network_domains
+
+The **aviatrix_network_domains** data source provides details about all Network Domains created by the Aviatrix Controller. Available as of provider version 2.23+.
+
+## Example Usage
+
+ ```hcl
+ # Aviatrix All Network Domains Data Source
+ data "aviatrix_network_domains" "foo" {
+   
```

Comment:
Small nit: You can leave out the newlines when there are no attributes. `data "aviatrix_network_domains" "foo" {}`

Reply:
have fixed